lisa_commands/response_helper.py

```diff
@@ -0,0 +1,56 @@
+"""
+A wrapper to apply additional fun options to the response we send to slack 
+(e.g. post with different name or icon image.)
+"""
+
+import datetime
+
+def apply_additional_message_options(body):
+    """
+    Lisa likes to get gussied up for special occasions. So we replace her
+    name and icon in the response that appears in slack.
+
+    body - response body (dict)
+    """
+
+    now = datetime.date.today()
+
+    # During the 15 Days leading up to Easter
+    days_till_easter = (calc_easter(now.year) - now).days
+    if days_till_easter >= 0 and days_till_easter < 15:
+        body['username'] ='Lucky Rabbit Lisa'
+        body["icon_url"] = 'https://raw.githubusercontent.com/Splyth/slack_bot/master/icons/Lisa_Easter.png'
```

Comment:
I'm getting a 404 when accessing these image urls

Reply:
They don't exist yet. They will once the merge to master is complete because those paths reference what they will be once merged. If you want to see them replace master with secret (the current branch)

Like so:
https://raw.githubusercontent.com/Splyth/slack_bot/secret/icons/Lisa_Easter.png
https://raw.githubusercontent.com/Splyth/slack_bot/secret/icons/Lisa_Christmas.png